DQM/HLTEvF/plugins/LumiMonitor.cc

```diff
@@ -267,6 +262,7 @@ void LumiMonitor::fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("pixelClusters", edm::InputTag("hltSiPixelClusters"));
   desc.add<edm::InputTag>("scalers", edm::InputTag("hltScalersRawToDigi"));
+  desc.add<edm::InputTag>("onlineMetaDataDigis", edm::InputTag("hltOnlineMetaDataDigis"));
   desc.add<std::string>("FolderName", "HLT/LumiMonitoring");
```

Comment:
could you change this to `folderName`, for consistency with all the other parameters that are lowercase ?

Reply:
Sure. For the 12_4_X backport, we might want to avoid this imho, to spare FOG from template migrations just because of this.

Reply:
OK.

How are the changes to the client configuration propagated to the online DQM ?

Reply:
That's a good question. So far, I only found one place, https://github.com/cms-sw/cmssw/pull/39859#discussion_r1005761983. I'll double-check, and hopefully DQM reviewers will know better than me.